recipes/xlrd/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.0.0" %}
+{% set name = "xlrd" %}
+{% set sha256 = "0ff87dd5d50425084f7219cb6f86bb3eb5aa29063f53d50bf270ed007e941069" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+
+test:
+  imports:
+
+about:
+  home: 'http://www.python-excel.org/'
+  license: "BSD-style"
```

Comment:
The licensee file is in the docs as a python module, which actually contains 2 different licenses. Probably because several parts of the code were written by 2 different people. However, they specifically state BSD in the setup.py and the webpage. No information about the clause though.